docs/docker-dev/docker-dev-setup-manual.md

```diff
@@ -0,0 +1,18 @@
+# Docker Development Environment Setup
+
+1. Install [Docker](https://docs.docker.com/get-docker/) if not already.
+2. On the terminal, run command `curl -o- https://raw.githubusercontent.com/opensearch-project/OpenSearch-Dashboards/main/dev-tools/install-docker-dev.sh | bash`. This should create a folder named `opensearch-dashboards-docker-dev` and it should contain two files: `docker-compose.yml` and `entrypoint.sh`.
+3. Open [VsCode](https://code.visualstudio.com/download). Install here if not already. Make sure VsCode has the extension `Dev Containers` and `Docker` installed. If not, go to Extensions tab, search and install them.
```

Comment:
We may want a little disclaimer at the top of the instructions that state the assumptions (using Docker and VS Code). Otherwise it's a bit surprising to only find out now.

```suggestion
3. Open VS Code or [install it](https://code.visualstudio.com/download), if it's not already. Make sure VS Code has the extensions `Dev Containers` and `Docker` installed. If not, go to Extensions tab, search and install them.
```

Reply:
Good idea, will add a paragraph at the top indicating this instruction uses docker and vscode.